arangod/Transaction/Manager.cpp

```diff
@@ -391,6 +395,29 @@ Result Manager::createManagedTrx(TRI_vocbase_t& vocbase, TRI_voc_tid_t tid,
                   std::string(TRI_errno_string(TRI_ERROR_ARANGO_DATA_SOURCE_NOT_FOUND)) +
                       ":" + cname);
       } else {
+#ifdef USE_ENTERPRISE
```

Comment:
seriously ?

Reply:
Tja, is there a better alternative for this?